module/product/features/product.feature

```diff
@@ -90,8 +104,7 @@ Feature: Product module
       """
       {
         "sku": "SKU_@@random_code@@",
-        "templateId": "@product_template@",
-        "categoryIds": ["@product_category@"]
+        "templateId": "@product_template@"
```

Comment:
Why You delete `categoryIds` ? 
Better add next one `Scenario`

Reply:
I create product with template and category in 70 line ;)